samples/msal-node-samples/msal-node-silent-flow/index.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+const express = require("express");
+const msal = require('@azure/msal-node');
+const { promises: fs } = require("fs");
+
+const SERVER_PORT = process.env.PORT || 3000;
```

Comment:
Can we make this default port 30662?

Reply:
Find with changing it, but if we let's make all node samples use the same port.

Reply:
I updated @pkanher617 offline, lets make it uniform in a `samples` only PR